package-lock.json

```diff
@@ -1,8 +1,3842 @@
 {
   "name": "zeebe-bpmn-moddle",
   "version": "0.12.2",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Are we using lock file version 2 now?

Reply:
I believe this is an oversight. The npm@8 PRs are still drafts.

Reply:
I was already preparing for tomorrows decision. Actually intentional. :wink: